addons/mail/static/src/web/thread_service_patch.js

```diff
@@ -106,7 +106,7 @@ patch(ThreadService.prototype, "mail/web", {
             type: "chatter",
         });
         if (resId === false) {
-            const tmpId = `virtual${this.nextId++}`;
+            const tmpId = this.messageService.getLastMessageId();
```

Comment:
The temporary message should not have same id as a genuine message.
You'd want to use `getNextTemporaryId()` instead ;)

Reply:
Yes, I noticed after my previous push and pushed again with `getNextTemporaryId`.